Sources/Views/MessageInputBar.swift

```diff
@@ -306,7 +306,7 @@ open class MessageInputBar: UIView {
     /// Layout the given UIStackView's
     ///
     /// - Parameter positions: The UIStackView's to layout
-    public func layoutStackViews(_ positions: [UIStackViewPosition] = [.left, .right, .bottom]) {
+    public func layoutStackViews(_ positions: [InputStackView.Position] = [.left, .right, .bottom, .top]) {
```

Comment:
I don't think we have a `.top` stackView yet? We should keep new features confined to a single PR. No partial implementations

Reply:
Got rid of all the .top refs for now